src/utils/buffer.cpp

```diff
@@ -78,6 +80,17 @@ void Buffer::initializeBuffer(unsigned int size, bool cyclic, bool output)
 				THROW_M2K_EXCEPTION("Buffer: Cannot create the RX buffer", libm2k::EXC_RUNTIME_ERROR, errno);
 			}
 		}
+		if (output) {
+                        LIBM2K_LOG(INFO,
+                                   libm2k::buildLoggingMessage({m_dev_name}, ("TX buffer created (" + std::to_string(size) +
+                                                                      " samples)").c_str()));
+		} else {
```

Comment:
Instead of the if else would something like this work in the build logging message
```cpp
libm2k::buildLoggingMessage({m_dev_name}, ((output ? "TX" : "RX") + "buffer created ...
```

Reply:
Modified.